src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/CellUpdate.cs

```diff
@@ -86,6 +86,22 @@ public CellUpdate(DbColumnWrapper column, string valueAsString)
         #region Properties
 
         /// <summary>
+        /// Converts the cell update to a DbCellValue
+        /// </summary>
+        public DbCellValue AsDbCellValue
```

Comment:
Not to sound grammar obsessed, but in this case "as" is used as a preposition :)
As such, I think it works fine as-is. (Yes, pun intended)